client/flutter/lib/pages/settings_page.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:WHOFlutter/constants.dart';
+import 'package:WHOFlutter/generated/l10n.dart';
+import 'package:WHOFlutter/components/list_of_items.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+/// TODO: Get all the strings from generated file
+
+class SettingsPage extends StatefulWidget {
+  @override
+  _SettingsPageState createState() => _SettingsPageState();
+}
+
+class _SettingsPageState extends State<SettingsPage> {
+  bool _switchValue = true;
```

Comment:
add ability to read this value from stored user preferences, or add a TODO comment + GitHub issue

Reply:
TODO please IMO

Reply:
I could fetch data from stored user preferences but as @advayDev1 suggested I'll add TODO for now.

Reply:
Yes, we need parallel work with analytics folks

Reply:
@advayDev1 & @hspinks  created #668 and #670 issues and pushed update with TODOs